libcodechecker/server/client_db_access_handler.py

```diff
@@ -1060,40 +1263,47 @@ def getDiffResultCount(self,
         Count the diff results.
         """
 
-        session = self.__session
-        base_line_hashes, new_check_hashes = \
-            self.__get_hashes_for_diff(session,
-                                       base_run_id,
-                                       new_run_id)
-
-        if diff_type == DiffType.NEW:
-            diff_hashes = list(new_check_hashes.difference(base_line_hashes))
-            if not diff_hashes:
-                return 0
-            run_id = new_run_id
-
-        elif diff_type == DiffType.RESOLVED:
-            diff_hashes = list(base_line_hashes.difference(new_check_hashes))
-            if not diff_hashes:
-                return 0
-            run_id = base_run_id
-
-        elif diff_type == DiffType.UNRESOLVED:
-            diff_hashes = list(base_line_hashes.intersection(new_check_hashes))
-            if not diff_hashes:
-                return 0
-            run_id = new_run_id
+        try:
```

Comment:
This implementation is very similar to `getDiffResultTypes`. Maybe it worth to add a TODO to refactor it.